music21/meter/base.py

```diff
@@ -489,6 +490,64 @@ def __init__(self, value: str = '4/4', divisions=None):
     def _reprInternal(self):
         return self.ratioString
 
+    def __eq__(self, other) -> bool:
+        '''
+        Establishes equality between two time signatures
```

Comment:
Dunder methods `__xxx__` don't display docs.  So put the basic information in the class documentation at the top, and just put corner case doctests here.

Reply:
Thanks @mscuthbert.

I understand that they don't display. Copying here from the practice seen elsewhere. E.g., `note.GeneralNote.__eq__` contains:

```
        '''
        General Note objects are equal if their durations are equal, and
        they have the same articulation and expression classes (in any order),
        and their ties are equal.
        '''
```

Should that comment (and any others like it) then move up to the class docs? Happy to implement (not in this PR, obv.)

Reply:
Actually in #1233 I made an exception for `__eq__`.

Reply:
Thanks @jacobtylerwalls.
This PR emulates @mscuthbert 's practice (discussed here and seen in #1466) of explaining equality in the class docs.
I'm obv. happy to implement either (i.e., as here or move back to `__eq__`).

Reply:
Sounds good. Just adding context to advertise we won't need a second refactor of the docstrings that survived #1466.

Reply:
Great. How does the rest of this PR look to you?